pkg/server/container_create_windows.go

```diff
@@ -325,12 +325,114 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 	g.AddAnnotation(annotations.SandboxID, sandboxID)
 
 	// Add OCI Mounts
-	for _, m := range config.GetMounts() {
+	err = c.addOCIMounts(&g, sandboxPlatform, config.GetMounts())
```

Comment:
Can we pass `plat` instead of `sandboxPlatform` here? Seems like we don't need to parse it again.

Reply:
we do have an explicit `"linux/amd64"` check against `sandboxPlatform`, rather than OS/Arch from parsed `plat`. I didn't want to break that behavior. Is it something I should worry about or checking against `plat.OS=="linux" && plat.Architecture=="amd64"` (or just `plat.OS=="linux"`) is safe enough?

Reply:
done!